apps/sms/views/conversation/js/conversation.js

```diff
@@ -265,6 +266,8 @@ var ConversationView = {
 
     // Bound methods to be detachables
     this.onMessageTypeChange = this.onMessageTypeChange.bind(this);
+
+    MessagingClient.init();
```

Comment:
__nit:__ maybe we should lazy load ```MessagingClient``` in both current monolithic and split views and initialize in shared/js/startup.js and views/conversation/startup.js instead? Looks like MessagingService doesn't include anything that maybe needed before visually-loaded for both setups...